GeoQuiz/app/src/main/java/com/bignerdranch/android/geoquiz/MainActivity.kt

```diff
@@ -0,0 +1,85 @@
+package com.bignerdranch.android.geoquiz
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.Button
+import android.widget.ImageButton
+import android.widget.TextView
+import android.widget.Toast
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var trueButton: Button
+    private lateinit var falseButton: Button
+    private lateinit var nextButton: ImageButton
+    private lateinit var questionTextView: TextView
+    private lateinit var prevButton: ImageButton
+
+    private val questionBank = listOf(
+        Question(R.string.question_australia, true),
+        Question(R.string.question_oceans, true),
+        Question(R.string.question_mideast, false),
+        Question(R.string.question_africa, false),
+        Question(R.string.question_americas, true),
+        Question(R.string.question_asia, true)
+    )
+
+    private var currentIndex = 0
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        trueButton = findViewById(R.id.true_button)
+        falseButton = findViewById(R.id.false_button)
+        nextButton = findViewById(R.id.next_button)
+        questionTextView = findViewById(R.id.question_text_view)
+        prevButton = findViewById(R.id.prev_button)
+
+        questionTextView.setOnClickListener {
+            currentIndex = (currentIndex + 1) % questionBank.size
```

Comment:
you could move the click listeners setup to a different method, not to overcrowd onCreate